Policies/Subscription/Azure_Subscription_SI_Classic_Resources/Readme.md

```diff
@@ -1,23 +1,23 @@
 ## Azure_Subscription_SI_Classic_Resources
 
 ### DisplayName 
-[Remove classic resources on a subscription](../../../Control%20coverage/Feature/SubscriptionCore.md)
+[Remove classic resources on a subscription](../../../Control%20coverage/Feature/SubscriptionCore.md#Azure_Subscription_SI_Classic_Resources)
 
 ### Required Policies
 Control can be covered with the below mentioned Azure policy:
 
 #### Policy Details
 
-Following policy can be used with 'Deny' effect to disallow users to create any classic resources in subscription (Greenfield Scenario).
+Following policy can be used with 'Audit' effect to disallow users to create any classic resources in subscription (Greenfield Scenario).
```

Comment:
Let's keep both statement as policy can work in both Deny and Audit mode.

Reply:
Adjusted the details according to effect.